src/lib/components/workflow/workflow-reset-form.svelte

```diff
@@ -2,35 +2,18 @@
   import Input from '$lib/holocene/input/input.svelte';
   import Option from '$lib/holocene/select/option.svelte';
   import Select from '$lib/holocene/select/select.svelte';
-  import { ResetReapplyType, ResetType } from '$lib/models/workflow-actions';
-  import type { WorkflowEvent } from '$lib/types/events';
+  import { ResetReapplyType } from '$lib/models/workflow-actions';
+  import { resetEvents } from '$lib/stores/events';
 
-  const DEFAULT_RESET_ID_HINT_TEXT =
-    'Only Workflow Events of type WorkflowTaskCompleted, WorkflowTaskFailed, or WorkflowTaskTimeout are supported.';
-
-  export let resetType: ResetType;
   export let resetReapplyType: ResetReapplyType = ResetReapplyType.Unspecified;
-  export let eventIdValid: boolean;
-  export let reason: string | undefined;
-  export let lastEvent: WorkflowEvent | undefined;
-  export let eventId: string | undefined = undefined;
-
-  let resetIdHintText: string = DEFAULT_RESET_ID_HINT_TEXT;
+  export let reason: string = '';
+  export let eventId: string = '';
 
-  const resetTypes = [
-    {
-      value: ResetType.FirstWorkflowTask,
-      label: 'First workflow task',
-    },
-    {
-      value: ResetType.LastWorkflowTask,
-      label: 'Last workflow task',
-    },
-    {
-      value: ResetType.EventId,
-      label: 'Event ID',
-    },
-  ];
+  $: {
+    if ($resetEvents && $resetEvents.length > 0) {
```

Comment:
Could `resetEvents` provide this? Why does this need to happen here?

Reply:
This is setting the default value on the prop `eventId` if there is a valid event that can be used as a reset point. The reason I do it this way is because this component may mount before `resetEvents` is populated with the valid events for a given workflow, so we need to check to make sure we have events before getting the ID of the first one. I can try to bake it into the store, but I think it makes sense here.

Reply:
I guess there's no real point in having a default value here, we should probably just disable the submit button until an event is selected.